spcs-three-tier/app/manifest.yml

```
@@ -8,6 +8,7 @@ configuration:
 
 artifacts:
   setup_script: setup.sql
+  readme: README.md
```

The file actually has a different case (in app/):
```suggestion
  readme: readme.md
```

---

done :D